### PR TITLE
feat(ip): Complete addr add/change/replace options

### PR DESCRIPTION
**Pull request by yedayak**

### CODE REVIEW

File: completions/ip [hunk 1]
```diff
@@ -186,7 +186,7 @@ _comp_cmd_ip()
                     elif [[ $prev == scope ]]; then
                         _comp_cmd_ip__iproute2_etc rt_scopes
                     else
-                        : # TODO
+                        _comp_compgen -- -W 'dev scope home nodad mngtmpaddr noprefixroute autojoin'
```
Comment (scop) [inline completions/ip:189]: Should we include all the words in `IFADDR`  as well? Ref https://github.com/iproute2/iproute2/blob/89210b9ec1c445ae963d181b5816d12a0cdafbb6/ip/ipaddress.c#L63-L66

**Review by scop — CHANGES_REQUESTED** [review]

File: completions/ip (hunk 1 above)
Comment (yedayak) [reply completions/ip:189]: Yeah we should, will do that soon

Comment (yedayak) [reply completions/ip]: Added them and pushed, should be ready

File: completions/ip [hunk 2]
```diff
@@ -181,13 +181,24 @@ _comp_cmd_ip()
...
+                            ;;
+                        broadcast | anycast | peer | metric)
+                            :
+                            ;;
+                        valid_lft | preferred_lft)
+                            _comp_compgen -- -W 'forever'
+                            ;;
+                        *)
+                            _comp_compgen -- -W 'anycast broadcast dev home label scope nodad metric mngtmpaddr
+			        noprefixroute valid_lft peer preferred_lft'
```
Comment (scop) [inline completions/ip:199]: Ubuntu 24.04 has `autojoin` in `CONFFLAG`

**Review by scop — CHANGES_REQUESTED** [review]
LGTM, just one small remark

File: completions/ip (hunk 2 above)
Comment (yedayak) [reply completions/ip]: Added

**Review by scop — APPROVED** [review]
Thanks!

**Pull request merged**